### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by nden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,19 @@
 This is a repository for the Roman Attribute Dictionary 
 
 
+![STScI Logo](docs/_static/stsci_logo.png)
```
Comment (eslavich) [inline README.md:7]: This will display as "STScI Logo" on the package's pypi page:

image

(may not be a problem, just thought I'd mention)

**Review by PaulHuwe — APPROVED** [review]
LGTM

**Review by ddavis-stsci — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (nden) [reply README.md]: Removed the logo. It's Hubble anyway.

**Pull request merged**